server/utils/httputils/utils.go

```diff
@@ -81,17 +67,55 @@ func WriteJSONStatus(w http.ResponseWriter, statusCode int, v interface{}) {
 }
 
 func WriteInternalServerError(w http.ResponseWriter, err error) {
-	WriteJSONError(w, http.StatusInternalServerError, "An internal error has occurred. Check app server logs for details.", err)
+	WriteError(w, http.StatusInternalServerError, err)
 }
 
 func WriteBadRequestError(w http.ResponseWriter, err error) {
-	WriteJSONError(w, http.StatusBadRequest, "Invalid request.", err)
+	WriteError(w, http.StatusBadRequest, err)
 }
 
 func WriteNotFoundError(w http.ResponseWriter, err error) {
-	WriteJSONError(w, http.StatusNotFound, "Not found.", err)
+	WriteError(w, http.StatusNotFound, err)
 }
 
 func WriteUnauthorizedError(w http.ResponseWriter, err error) {
-	WriteJSONError(w, http.StatusUnauthorized, "Unauthorized.", err)
+	WriteError(w, http.StatusUnauthorized, err)
+}
+
+const InLimit = 10 * (1 << 20)
+
+func ReadAndClose(in io.ReadCloser) ([]byte, error) {
+	defer in.Close()
+	return LimitReadAll(in, InLimit)
+}
+
+func LimitReadAll(in io.Reader, limit int64) ([]byte, error) {
+	if in == nil {
+		return []byte{}, nil
+	}
+	return ioutil.ReadAll(&io.LimitedReader{R: in, N: limit})
```

Comment:
nit pick: `ioutil` is deprecated 
```suggestion
	return io.ReadAll(&io.LimitedReader{R: in, N: limit})
```

Reply:
still needs change